decidim-meetings/app/controllers/decidim/meetings/widgets_controller.rb

```diff
@@ -9,7 +9,7 @@ class WidgetsController < Decidim::WidgetsController
       private
 
       def model
-        @model ||= Meeting.where(component: params[:component_id]).find(params[:meeting_id])
+        @model ||= Meeting.published.where(component: params[:component_id]).find(params[:meeting_id])
```

Comment:
I guess that we do not want to display any widgets from moderated or withdrawn meetings. 
```suggestion
        @model ||= Meeting.published.except_withdrawn.not_hidden.where(component: params[:component_id]).find(params[:meeting_id])
```

Reply:
Here we have a possible security issue when multi tenancy is involved. Reading the piece of code, it seems that is possible to access the widgets cross organization.

Reply:
Solved with 059fd6067a

Reply:
> I guess that we do not want to display any widgets from moderated or withdrawn meetings.

Moderated solved on 514689182db2fead7961efc6af094d961cb91d1d
Withdrawn solved on cb1b0aa4213d583c237c61f5427926cd98b5258c